packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -149,23 +149,18 @@ const BpkPhoneInput = (props: Props) => {
       className={getClassName('bpk-phone-input', wrapperProps.className)}
     >
       <div>
-        <BpkLabel
-          htmlFor={dialingCodeProps.id}
-          disabled={disabled}
-          // TODO: className to be removed
-          // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-          className={getClassName('bpk-phone-input__label')}
-        >
-          {dialingCodeProps.label}
-        </BpkLabel>
+        <div className={getClassName('bpk-phone-input__label')}>
+          <BpkLabel
+            htmlFor={dialingCodeProps.id}
+            disabled={disabled}
+          >
+            {dialingCodeProps.label}
+          </BpkLabel>
+        </div>
         {/* $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'. */}
         <BpkSelect
           {...commonProps}
           {...dialingCodeProps}
-          // TODO: className to be removed
-          // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-          className={getClassName(dialingCodeProps.className)}
-          wrapperClassName={getClassName(dialingCodeProps.wrapperClassName)}
```

Comment:
So on closer inspection the `dialingCodeProps` set the className and wrapperClassName which means these ones here are overrides for the default props. But also we don't detect that were setting className here as it's being set from the spread from `dialingCodeProps` above. There may be more cases of this which might have been missed.

Reply:
spread props should be caught by the _parent_ component so we are covered there (and if it's available on the public interface of the component there is nothing we can do either). 

"named" className props are potentially still problematic, however, there are far fewer of them and so I'm far less concerned.

Reply:
Ah, I reverted the `BpkInput` and `BpkSelect` changes and just left the `BpkLabel` ones in there as per the original Backpack audit doc.

Are we saying we are fine with the `BpkInput` and `BpkSelect` changes being included here?

Reply:
Lets leave the PR as is, is there another ticket for the BpkInput? If not, let's create one to see if anything can be done about the remaining `className` there or we move it to allow.

Reply:
I'll create another ticket for it, cannot see anything currently that matches 👍

Reply:
New ticket created for `BpkInput`.